Documentation/helm-ceph-cluster.md

```diff
@@ -52,18 +52,34 @@ The following tables lists the configurable parameters of the rook-operator char
 | --------------------- | -------------------------------------------------------------------- | ----------- |
 | `operatorNamespace`   | Namespace of the Rook Operator                                       | `rook-ceph` |
 | `configOverride`      | Cluster ceph.conf override                                           | <empty>     |
+| `ingress`             | Enable an ingress for the ceph-dashboard                             | `{}`        |
```

Comment:
for future-proofing, can this be called `dashboard.ingress` ?

Reply:
Agreed, we may also want an ingress for an object store, which would go under the objectstores section.

Reply:
Sure, I'll update

Reply:
Suggest the ingress for an objectStore is out of scope for this PR, I have used object stores for ceph before, but always in-cluster rather than exposing to external users.